src/getHumanAge.test.js

```diff
@@ -7,4 +7,52 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return an array', () => {
+    expect(getHumanAge(23, 23))
+      .toBeInstanceOf(Array);
+  });
+
+  test(`should return an array with two elements`, () => {
+    expect(getHumanAge(15, 15))
+      .toHaveLength(2);
+  });
+
+  test(`should return 0 if their age is 0`, () => {
+    expect(getHumanAge(0, 0))
+      .toEqual([0, 0]);
+  });
+
+  test(`should return 0
+    if both catAge and dogAge are less than or equal to 14`, () => {
```

Comment:
The test description and the expected result do not match. According to the rules of conversion, if cat's age and dog's age are less than or equal to 14, the human age is not 0.